### PR TITLE
#584: Upgrade Qulice to 0.16

### PR DESCRIPTION
**Pull request by essobedo**
Fix for https://github.com/yegor256/takes/issues/584

The goal of this PR is to fix all the issues raised by Qulice that are related to the upgrade from 0.15.4 to 0.16.

### CODE REVIEW

**Comment by davvd** [comment]
@essobedo Thanks, I will ask someone to do a code review here, thanks for your pull request

**Comment by davvd** [comment]
@longtimeago this pull request is for you,please review

**Comment by longtimeago** [comment]
@rultor merge

**Comment by essobedo** [comment]
@yegor256 please confirm

**Comment by yegor256** [comment]
@rultor merge

**Comment by essobedo** [comment]
@yegor256 _'pid' file is absent, something was broken_ should I report that as a bug in rultor project?

@yegor256 Could you please relaunch the merge?

**Comment by essobedo** [comment]
@longtimeago PR updated to fix the conflicts, please review

**Comment by longtimeago** [comment]
@rultor merge

**Comment by essobedo** [comment]
@yegor256 please confirm

**Comment by yegor256** [comment]
@rultor merge

**Comment by longtimeago** [comment]
@rultor try to merge ones more please

**Comment by essobedo** [comment]
@yegor256 Thank you. I have fixed the issue could you please confirm again? Thank you in advance

**Comment by essobedo** [comment]
@yegor256 please confirm

**Comment by essobedo** [comment]
@yegor256 please confirm

File: pom.xml [hunk 1]
```diff
@@ -313,12 +313,7 @@
                     <plugin>
                         <groupId>com.qulice</groupId>
                         <artifactId>qulice-maven-plugin</artifactId>
-                         <!--
-                         @todo #558:30min Upgrade Qulice to 0.16. Fix all checkstyle
-                          violations (mostly `AtclauseOrderCheck`). Fix or add puzzle for
-                          PMD, findbugs and other types of violations.
-                         -->
-                        <version>0.15.4</version>
+                        <version>0.16</version>
```
Comment (yegor256) [inline pom.xml]: @essobedo I think in scope of this ticket we should get rid of a few of the lines below. otherwise, your contribution is very small

**Comment by yegor256** [comment]
@essobedo I think this PR is rather small for a 30 minutes work. @longtimeago don't you think? we should remove at least some exclusions from that list, I believe.

**Comment by longtimeago** [comment]
@yegor256 I would say "yes" to you, but I see another puzzle  there (https://github.com/yegor256/takes/pull/585/files#diff-600376dffeb79835ede4a0b285078036L326), created especially to deal with exclusions.

**Comment by yegor256** [comment]
@longtimeago that's right, but this puzzle that we're solving here is saying that we have to upgrade to 0.16. we do upgrade, but leave a lot of exclusions... these puzzles overlap each other... anyway, OK, let's merge it

**Comment by yegor256** [comment]
@rultor merge

**Comment by longtimeago** [comment]
@rultor deploy

**Comment by yegor256** [comment]
@rultor deploy

**Comment by davvd** [comment]
@elenavolokhova review this ticket please, for compliance with our [quality rules](http://at.teamed.io/qa.html)

**Comment by davvd** [comment]
@rultor deploy now pls

**Comment by elenavolokhova** [comment]
@longtimeago 
According to our [quality rules](http://www.teamed.io/qa.html):
 > The code reviewer found at least three problems in the code.
 
 There were no issues found during review.
Please confirm that you will try to find at least three problems next time.

**Comment by longtimeago** [comment]
@elenavolokhova I confirm

**Comment by elenavolokhova** [comment]
@davvd Quality is acceptable here.

**Comment by davvd** [comment]
> @davvd Quality is acceptable here.

@elenavolokhova thanks for the QA review, we'll work better next time

**Comment by davvd** [comment]
@longtimeago just added 10 mins to @elenavolokhova (for QA), payment ID is `79095082`. **21 mins** sent to your balance (ID `79095106`), many thanks! It took 399 hours and 41 mins.. extra minutes for review comments (c=6). added +21 to your rating, now it is equal to [+6470](http://www.netbout.com/b/35024?open=rating)

**Pull request merged**